CMakeLists.txt

```diff
@@ -0,0 +1,176 @@
+cmake_minimum_required (VERSION 3.19)
```

Comment:
It would be silly to put a Msft copyright / MIT lic in a cmake file right?  Just checking.  I think it'd be silly.